src/abstract.jl

```diff
@@ -206,23 +206,39 @@ An alias for size.
 shape(op::AbstractLinearOperator) = size(op)
 
 """
-    hermitian(op)
     ishermitian(op)
 
 Determine whether the operator is Hermitian.
 """
-hermitian(op::AbstractLinearOperator) = op.hermitian
 ishermitian(op::AbstractLinearOperator) = op.hermitian
 
 """
-    symmetric(op)
+    Hermitian(op, uplo=:U)
+"""
+function Hermitian(op::AbstractLinearOperator, uplo::Symbol = :U)
+  ishermitian(op) || throw(LinearOperatorException("Operator is not Hermitian"))
+  return op
+end
+
+hermitian(A::AbstractLinearOperator, uplo::Symbol) = Hermitian(A, uplo)
```

Comment:
Maybe this should be marked as deprecated.

Reply:
How should I do this?

Reply:
https://docs.julialang.org/en/v1/base/base/#Base.@deprecate

Reply:
There's also a test: https://docs.julialang.org/en/v1/stdlib/Test/#Test.@test_deprecated

Reply:
Thanks for the hints, I've made the changes.